### PR TITLE
EOS-19732 : Ensure that Hare RPM is upgradeable

### PR DESCRIPTION
**Pull request by suvratjoshi**
Update Hare RPM version to 1.0.1 for LR2 to be testable.
This follows the versioning protocol as discussed within the
team and explained in this confluence page :
https://seagate-systems.atlassian.net/wiki/spaces/PUB/pages/424346235/26%2BVER%2BHare%2Bsource%2Bcode%2Bversioning?focusedCommentId=430440562#comment-430440562

The bumping of version numbers is decided, so that we can have
multiple versions and upgradation feature becomes testable now
onwards.

Signed-off-by: Suvrat Joshi <suvrat.joshi@seagate.com>

### CODE REVIEW

**Comment by suvratjoshi** [comment]
After this change, for testing :
1. I created the Hare RPM on my local VM and the RPM was correctly named with updated version 2.0.0.
2. Installed the RPM on VM and cluster was created successfully.
3. [Custom build](http://cortx-storage.colo.seagate.com/releases/cortx/github/integration-custom-ci/centos-7.8.2003/custom-build-1601/cortx_iso/) was created successfully using Jenkins and the created build had Hare rpm files correctly named with new version 2.0.0.

**Review by knekrasov — APPROVED** [review]

**Comment by mssawant** [comment]
@suvratjoshi , Summary of the PR does not really match the changes. Changing the  version in VERSION file to `2.0.0` is not really sufficient to make sure that Hare RPM is upgradeable right?
How about "Update Hare RPM version to `2.0.0` for LR2"?

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-1.0.0
+1.0.1
```
Comment (mssawant) [inline VERSION:1]: So we are not moving to 2.0.0 but bumping up minor version for every commit?

**Comment by mssawant** [comment]
@suvratjoshi, can you briefly describe why we are changing the version and how will it work going forward in the commit message and the PR description?

File: VERSION (hunk 1 above)
Comment (suvratjoshi) [reply VERSION:1]: Latest change is 1.0.1.
Yes. This we decided in a team discussion.

**Comment by suvratjoshi** [comment]
Addressed the comments and suggestion @mssawant.
Thank you!

File: VERSION (hunk 1 above)
Comment (mssawant) [reply VERSION:1]: Okay.

**Review by mssawant — APPROVED** [review]

**Review by knekrasov — APPROVED** [review]

**Pull request merged**